build.psm1

```diff
@@ -1838,84 +1859,6 @@ function Start-DevPowerShell {
     }
 }
 
-
-<#
-.EXAMPLE
-PS C:> Copy-MappedFiles -PslMonadRoot .\src\monad
-
-copy files FROM .\src\monad (old location of submodule) TO src/<project> folders
-#>
-function Copy-MappedFiles {
```

Comment:
Does this belong in a distinct change for another PR? Maybe also removing the json files and other functions like `Get-Mappings`?